### DISCUSSION TITLE
Feature Request: Plex Watchlist Integration

### DISCUSSION THREAD

**Discussion by Caesar679** [sct/overseerr#2712] [Feature Requests]
It would be great if the new native watchlist feature in Plex could automatically send a request to overseer to find movies. This seems very difficult to achieve but incredibly useful. Thanks!

**Comment by stroodle96** [comment]
Agreed this would be an awesome feature. Radar and Sonar already have this feature, however you need to have each of your Plex users authenticate, so you would have to give them access to your arr's or ask for their credentials. Overseer seems like the perfect solution to automatically pull this in for them since the already login to overseerr with Plex credentials.

**Reply by stroodle96** [reply]
Looks like someone already opened #2678 to request this.
